app/services/importers/party_html.rb

```diff
@@ -13,8 +13,10 @@ def self.perform(oscn_id)
     end
 
     def perform
-      county_name = @party.court_cases.first.county.name
-      data = OscnScraper::Requestor::Party.fetch_party(county_name, oscn_id)
+      sample_courtcase = @party.court_cases.first
+      county_name = sample_courtcase.county.name
+      case_number = sample_courtcase.case_number
+      data = OscnScraper::Requestor::Party.fetch_party(county_name, oscn_id, case_number)
```

Comment:
This assumes that Parties never cross counties. I think it is a safe assumption, but I don't remember ever checking after we expanded beyond OK county

Reply:
I don't think it would actually matter if there were because I'd assume the party data for all counties would be the same.